src/main/java/com/stuypulse/robot/RobotContainer.java

```diff
@@ -191,7 +191,10 @@ private void configureOperatorBindings() {
             .whileTrue(
                 new LEDSet(LEDColor.RED)
                     .andThen(new ManagerValidateState())
-                    .andThen(new ArmReady()));
+                    .andThen(new ArmReady())
+                    .alongWith(new IntakeAcquire()).withTimeout(0.2)
```

Comment:
it looks like this timeout went onto the whole commanf group, when it should just be only on the intake acquire, ALSO you need to stop the intake after this wait command.

so maybe it should be a group of 
intake acquire, wait 0.2 seconds, intake stop